toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,11 +2,36 @@
 
 # Required binaries:
 # rpm and dnf
+sodiff_script_error=false
+while getopts "r:f:v:o:e:" opt; do
+    case $opt in
+        r) rpms_folder="$OPTARG";;
+        f) repo_file_path="$OPTARG";;
+        v) mariner_version="$OPTARG";;
+        o) sodiff_out_dir="$OPTARG";;
+        e) sodiff_script_error="$OPTARG";;
+    esac
+done
+
+if [[ -z "$rpms_folder" ]]; then
+    echo "INVALID ARGUMENT: RPMS_FOLDER is empty"
+    exit 1
+fi
+
+if [[ -z "$repo_file_path" ]]; then
+    echo "INVALID ARGUMENT: REPO_FILE_PATH is empty"
+    exit 1
+fi
```

Comment:
Might be worth mentioning the flag, which is necessary to pass this value. Right now it's hard to say that `RPMS_FOLDER` is set by `-r`.

Reply:
Added error messages for each command option.